package.json

```diff
@@ -49,7 +49,7 @@
   ],
   "activationEvents": [
     "onStartupFinished",
-    "onView:continueGUIView"
+    "onView:graniteGUIView"
```

Comment:
is the continueGUIView string renamed graniteGUIView by the build? Else stuff like that would break:

> vscode.commands.executeCommand("continue.continueGUIView.focus");

Reply:
see https://github.com/Granite-Code/continue-for-granite/blob/granite/customization/extensions/vscode/src/ContinueGUIWebviewViewProvider.ts#L18

Reply:
Yes, it's renamed in our ESBuild script:

https://github.com/Granite-Code/granite-code/blob/4f230f8569998e72d1a010ac768e0b3d33171e0d/scripts/build.js#L74